sdk/eventhub/azure-eventhub/azure/eventhub/_eventprocessor/_eventprocessor_mixin.py

```diff
@@ -99,7 +99,9 @@ def _context(self, links=None):
         if span_impl_type is None:
             yield
         else:
-            child = span_impl_type(name="Azure.EventHubs.process", kind=SpanKind.CONSUMER, links=links)
```

Comment:
NOTE: ALL FILES IN _EVENTPROCESSOR FOLDER CAN BE IGNORED. No actual changes here.